static/js/recent_topics_util.js

```diff
@@ -29,3 +29,12 @@ export function is_in_focus() {
 export function get_topic_key(stream_id, topic) {
     return stream_id + ":" + topic.toLowerCase();
 }
+
+export function get_key_from_message(msg) {
+    if (msg.type === "private") {
+        return msg.to_user_ids;
```

Comment:
Added this explanatory comment:

        // The to_user_ids field on a private message object is a                                        
        // string containing the user IDs involved in the message in                                     
        // sorted order.